src/vm/exceptionhandling.h

```diff
@@ -389,15 +389,15 @@ class ExceptionTracker
 
     bool IsStackOverflowException();
 
-#ifdef FEATURE_PAL
+#if defined(FEATURE_PAL) && !defined(CROSS_COMPILE)
```

Comment:
@janvorli I think exception handling such as stack unwind is needless in cross-architecture component. Is there have any places uses this in cross-architecture component?